test/JDBC/expected/babel_context_info-vu-verify.out

```diff
@@ -0,0 +1,21 @@
+SET CONTEXT_INFO 12345678
+GO
+
+EXEC GET_CONTEXT_INFO
+GO
+~~START~~
+varbinary
+00BC614E00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+~~END~~
+
+
+EXEC SET_CONTEXT_INFO 0x12345678
+GO
+
+SELECT CONTEXT_INFO()
+GO
+~~START~~
+varbinary
+1234567800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

Comment:
The returned value should start with '0x'

Reply:
Yeah, 0x0000000A is the format in sqlcmd, 0000000A is the format in JDBC.